plugins/base_plugin.py

```
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import click
```

Ruff (0.8.2)

`click` imported but unused

Remove unused import: `click`

(F401)